microsoft/testsuites/vm_extensions/runtime_extensions/common.py

```diff
@@ -96,6 +97,20 @@ def retrieve_storage_blob_url(
     )
     is_public_container = container_name.endswith("-public")
     blob_data = script or f"touch {test_file}"
+    if is_public_container:
```

Comment:
Move priority should be enough, it doesn't need to mark them as skippped.

Reply:
updated